services/common/src/components/projectSummary/AuthorizationsInvolved.tsx

```diff
@@ -454,8 +628,12 @@ export const AuthorizationsInvolved = () => {
                                   />
```

Comment:
This is from a previous PR it seems, but just noticed a typo on line 622 `Authorixation` should be `Authorization` and `FrountCounter BC` should be `FrontCounter BC`

Reply:
Corrected.